include/soci/oracle/soci-oracle.h

```diff
@@ -278,6 +279,77 @@ struct oracle_session_backend : details::session_backend
             " where table_name = :t";
     }
     
+    virtual std::string create_column(const std::string & columnName, data_type dt,
+        int precision, int scale)
+    {
+        //  Oracle-specific SQL syntax:
+        
+        std::string res = columnName + " ";
```

Comment:
Perhaps we should always put quotes around the column name to avoid possible conflicts with SQL keywords?